### PR TITLE
2.6 create table of teachers maslova

### PR DESCRIPTION
**Pull request by maslovakat**

### CODE REVIEW

File: src/migrations/20200415143335-create-create-table-of-teachers.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { QueryInterface, DataTypes } from 'sequelize';
+import { Teachers } from '../models/create_table_of_teachers';
+
+export async function up(query: QueryInterface) {
+  try {
```
Comment (Alan18081) [inline src/migrations/20200415143335-create-create-table-of-teachers.ts:5]: We don't need try ... catch blocks here. Just return the calling of createTable method

File: src/migrations/20200415143335-create-create-table-of-teachers.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      },
+      deletedAt: DataTypes.DATE,
+    });
+  } catch (e) {
+    return Promise.reject(e);
+  }
+}
+
+export async function down(query: QueryInterface) {
+  try {
```
Comment (Alan18081) [inline src/migrations/20200415143335-create-create-table-of-teachers.ts:58]: We don't need try ... catch blocks here. Just return the calling of dropTable method

File: src/models/create_table_of_teachers.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { DataTypes, FindOptions, Model, ModelCtor, Sequelize } from 'sequelize';
+
+export interface ITeacher {
```
Comment (Alan18081) [inline src/models/create_table_of_teachers.ts:3]: Why do we need this interface here?

File: src/modules/teachers/teachers.model.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+  public id!: number;
+  public firstName: string;
+  public lastName: string;
+  public email: string;
+  public password: string;
+  public isAdmin: boolean;
+  public createdAt: Date;
+  public updatedAt: Date;
+  static init: any;
```
Comment (Alan18081) [inline src/modules/teachers/teachers.model.ts:16]: What is the purpose of this property?

Comment (maslovakat) [reply src/modules/teachers/teachers.model.ts]: fixed

**Pull request merged**